ui/component/claimList/view.jsx

```diff
@@ -61,6 +61,9 @@ type Props = {
   unavailableUris?: Array<string>,
   showMemberBadge?: boolean,
   inWatchHistory?: boolean,
+  onHidden: string,
+  hasPremiumPlus: ?boolean,
+  adBlockerFound: boolean,
```

Comment:
Remove `hasPremiumPlus` and `adBlockerFound`
Remove `hasPremiumPlus` usage down below as the logic is now at the caller side.